internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/router/TestRead.java

```diff
@@ -753,4 +836,157 @@ public void testRequestUsageMetric() {
     assertEquals(metricsRepository.metrics().get(requestUsageMetric).value(), 1.0d);
     assertEquals(metricsRepository.metrics().get(badRequestMetric).value(), 1.0d);
   }
+
+  @Test(timeOut = 10 * Time.MINUTES_PER_HOUR)
```

Comment:
DO we really need 10 mins to run this test?

Reply:
nope. Used that to test out the time out which no longer happens